packages/icons/src/library/post-excerpt.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const postExcerpt = (
+	<SVG fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
```

Comment:
I'm not sure about these clip and fill rules. I don't really know if they are needed.